treeherder/webapp/api/jobs.py

```diff
@@ -91,10 +95,20 @@ def list(self, request, project, jm):
         - return_type (dict)
         """
         MAX_JOBS_COUNT = 2000
+        EARLIEST_LAST_MODIFIED = datetime.datetime.utcnow() - \
```

Comment:
I would set this to datetime.timedelta(minutes=30), then do the actual calculation below. As it is, it looks like a dynamically calculated constant, which seems weird.